test/clustering/test_clustering.py

```diff
@@ -0,0 +1,70 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2020 Aibolit
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+from pathlib import Path
+# flake8: noqa
+from unittest import TestCase
+
+from aibolit.ast_framework import AST
+from aibolit.utils.ast_builder import build_ast
+from aibolit.extract_method_baseline.extract_semantic import (
+    extract_method_statements_semantic,
+    StatementSemantic,
+)
+from aibolit.clustering.clustering import _reprocess_dict, process_statement
+
+
+class ClusteringTestCase(TestCase):
+      current_directory = Path(__file__).absolute().parent
+
+      def article_test(self):
```

Comment:
To automatically call this method name it with `test_` in the beginning.

Reply:
FIxed